### DISCUSSION TITLE
can otomad maker create chord visuals

### DISCUSSION THREAD

**Discussion by MikaelSunflowerseed** [otomad/OtomadHelper#61] [Q&A] [answered]
if my midi channel has chords,can otomad maker spread them out into 3 boxes or doess it stack them on each other

**Comment by otomad** [comment, marked as answer]
Yes, it can spread them out into 3 tracks.

You can also use "Auto Layout Tracks - Grid Layout" feature. To quickly make the tracks into the grids.
